pumps/kafka.go

```diff
@@ -119,8 +124,8 @@ func (k *KafkaPump) WriteData(data []interface{}) error {
 	return nil
 }
 
-func (k *KafkaPump) write(messages []kafka.Message) error{
+func (k *KafkaPump) write(messages []kafka.Message) error {
 	kafkaWriter := kafka.NewWriter(k.writerConfig)
 	defer kafkaWriter.Close()
```

Comment:
Are you sure we should close connection after every write call to kafka?

Reply:
In theory yes, with the defer operation when go out of write method kafka client is closed

Reply:
@letzya My previous version are without close conection and you ask me to close the connection. The only way i see currently with the pump is open and close connection on every batch. As i can see in other pump connection is opened for every batch. Then i assume we don't have any other way to define the close connection instead of open&close every batch.

Reply:
I need to check that.